elastic/kibana
# Add alertDetailsUrl to the Uptime rule types

Closes #158661

## Summary

This PR adds alertDetailsUrl to the Uptime rule types.

|Alert|Recovered|
|---|---|
|![image](https://github.com/elastic/kibana/assets/12370520/9adbcb71-ae6a-4a7a-9e2b-9a62b4c7e97e)|![image](https://github.com/elastic/kibana/assets/12370520/2605d32b-e234-49dc-84c8-6b573404dcaa)|

It also provides the alertDetailsUrl in the action variables:

<img width="920" alt="Screenshot 2023-05-31 at 14 54 06" src="https://github.com/elastic/kibana/assets/12370520/5d3a8195-ce8f-4ee9-a719-e4d553d63cff">

## 🧪 How to test
- Ensure that `server.publicBaseUrl` is configured in kibana.dev.yml
- Create an uptime rule
- Add an action to this rule and specify `context.alertDetailsUrl` variable there 
- After the alert is triggered, open the value of `alertDetailsUrl` in the browser
     - you should be able to see the related alert on the `Observability > Alerts` page
